src/node-backend/routes/classroom.js

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import * as classroomController from "../controller/classroom-controller.js";
+
+import { JoiningStudent } from "../models/domain-objects.js";
+export const router = express.Router();
+
+router.post('JoiningStudent', async function (req, res) {
+    try {
+        const {username} = req.body;

```

Comment:
this line is where you receive the data from the frontend, the req.body (data that are attached to the request from the frontend) contains all the variables we need, we just need to deconstruct them. So if you want to have a classId and a list of student names for example, you'd write
`const {classId, studentNames} = req.body;`
or if you want to have a classname and a teacher's full name,
`const {className, teacherName} = req.body;`